### PR TITLE
Fixes 305: option added to disable the repository in Yaydoc

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

I have added option to disbable yaydoc to the repository
## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: public/scripts/socket-dashboard.js [hunk 1]
```diff
@@ -77,4 +79,6 @@ $(function() {
       return false;
     }
   });
+
+  $("")
```
Comment (imujjwal96) [inline public/scripts/socket-dashboard.js]: what's this for?

**Comment by imujjwal96** [comment]
@sch00lb0y Can you please share a deployment link?

**Comment by sch00lb0y** [comment]
https://demo08.herokuapp.com

File: public/scripts/socket-dashboard.js [hunk 2]
```diff
@@ -45,7 +45,9 @@ $(function() {
     'registration_successful': "Registration successful! Hereafter Documentation will be pushed to the GitHub pages on each commit.",
     'registration_already': "This repository has already been registered.",
     'registration_failed': "Failed to register repository to Yaydoc!",
-    'registration_unauthorized': "You do not have admin permission for this repository."
+    'registration_unauthorized': "You do not have admin permission for this repository.",
+    'enabled_successful': "Yaydoc enabled successfully to the repository",
```
Comment (pri22296) [inline public/scripts/socket-dashboard.js]: *for this repository

**Comment by imujjwal96** [comment]
@sch00lb0y the callback points to localhost

**Comment by sch00lb0y** [comment]
@imujjwal96 I was testing. I have updated the callback. please check now

**Comment by imujjwal96** [comment]
I think we should follow an approach similar to deleting repository(Asking for a confirmation). @pri22296 What do you think?

**Comment by pri22296** [comment]
@imujjwal96 A `are you sure you want to disable this repo?` with a `yes` and `no` button would be fine.

**Comment by sch00lb0y** [comment]
@pri22296 @imujjwal96 I have updated https://demo08.herokuapp.com

**Comment by imujjwal96** [comment]
No action on enable/disable
image

**Comment by pri22296** [comment]
@sch00lb0y I don't think a confirmation is needed when enabling a repo.

**Comment by sch00lb0y** [comment]
@pri22296 @imujjwal96 updated http://demo08.herokuapp.com/

**Comment by imujjwal96** [comment]
Take inspiration from this modal
image

**Comment by imujjwal96** [comment]
Looks good otherwise

**Comment by sch00lb0y** [comment]
@imujjwal96 updated.

File: views/dashboard.jade [hunk 3]
```diff
@@ -117,3 +115,18 @@ block content
...
+    .modal-dialog
+      .modal-content
+        form#disableForm(action="/repository/disable" method="post")
+          .modal-header
+              input#disableInput(type="hidden", name="repository")
+              button.close(type='button', data-dismiss='modal') ×
+              h4.modal-title Are you sure?
+          .modal-body
+                p
+                | are you sure you want to disable this repo?
```
Comment (sch00lb0y) [inline views/dashboard.jade]: updated

**Pull request merged**